src/AbstractProxyFixer.php

```diff
@@ -53,6 +61,23 @@ public function getPriority()
     /**
      * {@inheritdoc}
      */
+    public function supports(\SplFileInfo $file)
+    {
+        return $this->proxyFixer->supports($file);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setWhitespacesConfig(WhitespacesFixerConfig $config)
+    {
+        parent::setWhitespacesConfig($config);
```

Comment:
right, i forgot that parent::setWC is checking the interface and raise exception, if not available
:+1: